tests/phpunit/payment-methods/class-wc-stripe-upe-payment-gateway-test.php

```
@@ -4603,4 +4603,386 @@ public function test_add_email_currency_conversion_notice_outputs_plain_text_for
+	 * Test that statement_descriptor is set in payment_information when using the intent update path.
+	 *
+	 * Note: The Stripe /confirm endpoint does not accept statement_descriptor, so the intent
+	 * controller intentionally does not forward it. This test verifies the gateway still sets
+	 * the value in payment_information for the update path.
+	 */
+	public function test_statement_descriptor_set_in_payment_info_on_intent_update() {
+		$order    = WC_Helper_Order::create_order();
+		$order_id = $order->get_id();
+
+		// Configure a local statement descriptor.
+		$stripe_settings                         = WC_Stripe_Helper::get_stripe_settings();
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Helper' in method 'test_statement_descriptor_set_in_payment_info_on_intent_update'. (undefined)

(StaticAccess)